packages/zowe-explorer/src/job/ZoweJobNode.ts

```diff
@@ -211,7 +231,16 @@ export class Job extends ZoweTreeNode implements IZoweJobTreeNode {
         return this.getParent() ? this.getParent().getSessionNode() : this;
     }
 
+    set tooltip(newTooltip: string) {
+        if (newTooltip !== undefined) {
+            this._tooltip = newTooltip;
+        }
+    }
```

Comment:
if (newTooltip) {
     this._tooltip = newTooltip;
  }
I think this check is saver against any [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) value.

Reply:
addressed in commit https://github.com/zowe/vscode-extension-for-zowe/pull/1920/commits/1889233360272455accc66bbba7c913a16d6a1aa